scripts/cubic/bootstrap-fmriprep.sh

```diff
@@ -166,48 +204,50 @@ cd ds
 # this remote is never fetched, it accumulates a larger number of branches
 # and we want to avoid progressive slowdown. Instead we only ever push
 # a unique branch per each job (subject AND process specific name)
-git remote add outputstore "$pushgitremote"
+git remote add outputstore "\$pushgitremote"
 
 # all results of this job will be put into a dedicated branch
-git checkout -b "${BRANCH}"
+git checkout -b "\${BRANCH}"
 
 # we pull down the input subject manually in order to discover relevant
 # files. We do this outside the recorded call, because on a potential
 # re-run we want to be able to do fine-grained recomputing of individual
 # outputs. The recorded calls will have specific paths that will enable
 # recomputation outside the scope of the original setup
-datalad get -n "inputs/data/${subid}"
+datalad get -n "inputs/data/\${subid}"
 
 # Reomve all subjects we're not working on
-(cd inputs/data && rm -rf `find . -type d -name 'sub*' | grep -v $subid`)
+(cd inputs/data && rm -rf `find . -type d -name 'sub*' | grep -v \$subid`)
 
 # ------------------------------------------------------------------------------
 # Do the run!
-
+# TODO: Be sure the actual path to the fmriprep container is correct
+# TODO FIX path!!
+echo Before running datalad run
 datalad run \
     -i code/fmriprep_zip.sh \
-    -i inputs/data/${subid} \
+    -i inputs/data/\${subid} \
     -i inputs/data/*json \
-    -i pennlinc-containers/.datalad/environments/fmriprep-20-2-3/image \
+    -i containers/images/bids/bids-fmriprep--${VERSION}.sing \
     --explicit \
-    -o ${subid}_fmriprep-20.2.3.zip \
-    -o ${subid}_freesurfer-20.2.3.zip \
-    -m "fmriprep:20.2.3 ${subid}" \
-    "bash ./code/fmriprep_zip.sh ${subid}"
+    -o \${subid}_fmriprep-${VERSION}.zip \
+    -o \${subid}_freesurfer-${VERSION}.zip \
```

Comment:
yarik said this flag can take folders. perhaps we can change this to accept a folder and thus not zip the data.

Reply:
I can ask him, but the documentation says file, can use a wildcard, but I believe can't use path to a folder: http://docs.datalad.org/en/stable/generated/man/datalad-run.html#o-path-output-path

Reply:
@yarikoptic - could you please comment on this?

Reply:
could be a directory. Sent a https://github.com/datalad/datalad/pull/6416